File: Api.Server/src/Services/Center/Merchant/Dto/QueryMerchantPagedInput.cs
namespace Fast.Center.Service.Merchant.Dto;

/// <summary>
/// <see cref="QueryMerchantPagedInput"/> 获取商户号分页列表输入
/// </summary>
public class QueryMerchantPagedInput : PagedInput
{
    /// <summary>
    /// 商户号类型
    /// </summary>
    public PaymentChannelEnum? MerchantType { get; set; }
}

File: Web.Admin/src/api/services/merchant/models/QueryMerchantPagedInput.ts
import { PagedInput } from "fast-element-plus";
import { PaymentChannelEnum } from "@/api/enums/PaymentChannelEnum";

/**
 * Fast.Center.Service.Merchant.Dto.QueryMerchantPagedInput 获取商户号分页列表输入
 */
export interface QueryMerchantPagedInput extends PagedInput  {
  /**
   * 
   */
  merchantType?: PaymentChannelEnum;
}

